rgw/v2/tests/s3cmd/reusable.py

```diff
@@ -56,6 +57,32 @@ def upload_file(bucket_name):
     return file_name
 
 
+def download_file(bucket_name, remote_file_name, local_file_name=None):
+    """
+    Downloads file from the bucket
+    Args:
+        bucket_name(str): Name of the bucket
+        remote_file_name(str): Name of the remote file
+        local_file_name(str): Name of the local file to be set
+    Returns: Name of the downloaded file
+    """
+    if local_file_name is None:
+        local_file_name = "test_s3cmd.txt"
+
+    download_file_method = S3CMD(operation="get")
+    remote_s3_path = "s3://{}/{}".format(bucket_name, remote_file_name)
```

Comment:
Opt to use f-string format instead of `format`... i.e. `remote_s3_path = "s3://{bucket_name}/{remote_file_name}"`

Reply:
Done